### ISSUE TITLE
Use cases and examples?

### ISSUE THREAD

**Issue by cbluth** [skyllo/peer-lite#31]
Seeing this on hn, and i was interested... But i don't know what it does (eli5), or why someone might want it

**Comment by skyllo** [comment]
On a basic level this project is a wrapper around [RTCPeerConnection](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) which allows peer-to-peer connections to share audio, video and data. The normal use-case is to provide peer-to-peer video calls with the browser using a signalling server to setup the initial connection.

You can find a working example of this in the [example folder](https://github.com/skyllo/peer-lite/tree/master/example). There are also some [great guides](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling) on MDN that can help explain WebRTC better. Hope this helps!
